java/garbage-collector/src/test/java/sleeper/garbagecollector/GarbageCollectorIT.java

```diff
@@ -71,14 +75,22 @@
 
 @Testcontainers
 public class GarbageCollectorIT {
-
+    private static final Schema TEST_SCHEMA = getSchema();
+    private static final String TEST_TABLE_NAME = "test-table";
+    private static final String TEST_TABLE_NAME_1 = "test-table-1";
+    private static final String TEST_TABLE_NAME_2 = "test-table-2";
+    private static final int DEFAULT_BATCH_SIZE = 2;
+    private static final int DEFAULT_GC_DELAY = 10;
+    private static final Instant CURRENT_TIME = Instant.parse("2023-06-28T13:46:00Z");
+    private static final Instant TIME_AFTER_DELAY = CURRENT_TIME.minus(Duration.ofMinutes(DEFAULT_GC_DELAY + 1));
```

Comment:
It might be easier to read if we inline these constants in the tests they're relevant to.